Tasks/DeployAzureResourceGroup/SPNCreation.ps1

```diff
@@ -13,9 +16,9 @@ param
 #Initialize
 $ErrorActionPreference = "Stop"
 $VerbosePreference = "SilentlyContinue"
-$userName = $env:USERNAME
+#$userName = $env:USERNAME
 $newguid = [guid]::NewGuid()
-$displayName = [String]::Format("VSO.{0}.{1}", $userName, $newguid)
+$displayName = [String]::Format("VSO.{0}.{1}", $appName, $newguid)
 $homePage = "http://" + $displayName
 $identifierUri = $homePage
```

Comment:
Can you also do the following and update the PR:
a) Update patch filed in the task.json. This is something we do for every change in Task files.
b) please do a "gulp build". This will generate appropriate task.loc.json. Include that in your PR.
c) Please run "gulp test" once and ensure it is successful.